enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -499,6 +499,30 @@ Dashboards that support editing if a user has permissions for this, could be emb
   but without its cards - at this stage dashboard title, tabs and cards grid is rendered, but cards content is not yet
   loaded.
 
+### Creating a Question
+
+With the `NewQuestion` component, you can create a new question from scratch using the Metabase Notebook Editor. 
+
+#### Parameters
```

Comment:
These are universally called "props".
Or is it supposed to be React-agnostic?

```suggestion
#### Props
```

---

_It's the first time I'm seeing this README, nice!_

Reply:
We're going to do a general refactor of the README but right now our convention is `Parameters`